src/domain/help/utils.ts

```diff
@@ -0,0 +1,82 @@
+import { FormikErrors, FormikTouched } from 'formik';
+import forEach from 'lodash/forEach';
+import set from 'lodash/set';
+import { User } from 'oidc-client';
+import { scroller } from 'react-scroll';
+import * as Yup from 'yup';
+
+import { CONTACT_FORM_SELECT_FIELDS, initialValues } from './constants';
+import { ContactFormFields } from './types';
+
+export const showErrors = ({
+  error,
+  setErrors,
+  setTouched,
+}: {
+  error: Yup.ValidationError;
+  setErrors: (errors: FormikErrors<ContactFormFields>) => void;
+  setTouched: (
+    touched: FormikTouched<ContactFormFields>,
+    shouldValidate?: boolean
+  ) => void;
+}): void => {
+  /* istanbul ignore else */
+  if (error.name === 'ValidationError') {
+    const newErrors = error.inner.reduce(
+      (acc, e: Yup.ValidationError) =>
+        set(acc, e.path ?? /* istanbul ignore next */ '', e.errors[0]),
+      {}
+    );
+    const touchedFields = error.inner.reduce(
+      (acc, e: Yup.ValidationError) =>
+        set(acc, e.path ?? /* istanbul ignore next */ '', true),
+      {}
+    );
+
+    setErrors(newErrors);
+    setTouched(touchedFields);
+  }
+};
```

Comment:
This set errors but also sets some field to touched state. Maybe some comment/explanation would be nice

Reply:
Added a comment